Source/WebGPU/WebGPU/ComputePassEncoder.mm

```diff
@@ -55,15 +55,16 @@
 
 void ComputePassEncoder::dispatch(uint32_t x, uint32_t y, uint32_t z)
 {
-    UNUSED_PARAM(x);
-    UNUSED_PARAM(y);
-    UNUSED_PARAM(z);
+    ASSERT(!(x % m_threadsPerThreadgroup.width) && !(y % m_threadsPerThreadgroup.height) && !(z % m_threadsPerThreadgroup.depth));
+    if ((x % m_threadsPerThreadgroup.width) || (y % m_threadsPerThreadgroup.height) || (z % m_threadsPerThreadgroup.depth))
+        return;
+
+    [m_computeCommandEncoder dispatchThreads:MTLSizeMake(x / m_threadsPerThreadgroup.width, y / m_threadsPerThreadgroup.height, z / m_threadsPerThreadgroup.depth) threadsPerThreadgroup:m_threadsPerThreadgroup];
```

Comment:
I think this is wrong for 2 reasons:

1. DispatchThreads isn't available on all devices IIRC
2. I believe the arguments indicate the number of workgroups to launch, not the number of threads to launch

Reply:
Great catch! Totally meant to write `dispatchThreadgroups`

Reply:
Please fixup (or, ideally, remove) the math above this, to make it appropriate for dispatchThreadgroups